manifests/p/prefix-dev/pixi/0.9.1/prefix-dev.pixi.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: prefix-dev.pixi
+PackageVersion: 0.9.1
+InstallerLocale: en-US
+InstallerSwitches:
+  Custom: -norestart
+UpgradeBehavior: install
+ReleaseDate: 2023-11-29
+AppsAndFeaturesEntries:
+- DisplayName: pixi
+  Publisher: pixi contributors

```

```suggestion
  Publisher: prefix-dev
```
Maybe this fits better? Not sure though

---

Hmmm .. both is fine with me

---

Komac filled this value in automatically. Prefix-dev does seem more in line to me.